app/views/_includes/candidate-feedback.njk

```diff
@@ -23,7 +23,7 @@
       Something you did
     </h3>
 
-    <ul class="govuk-list govuk-list--bulle app-list--spaced">
+    <ul class="govuk-list govuk-list--bullet app-list--spaced">
```

Comment:
The typo was on purpose I think in order to not use bullets. Please just kill/double check

Reply:
We simply need to remove `govuk-list--bullet` if we don't want to use bullets. No need for a typo.

Reply:
Wow, what a thing to say publically. Are you trying to put me to shame? :D

It was out of laziness. I can type ‘t’ or I can type the whole thing. :D